### DISCUSSION TITLE
BloatynosyAI 2.0

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#133] [Announcements]
**Today, BloatynosyAI enters a new era with the first preview version of version 2.0.**
This marks a major release with a hybrid web app foundation _(which was developed for the CoMachina app. Bloatynosy may potentially merge with CoMachina, but this decision will be made together)._ 

This version will be given the additional identifier "AI" by me and will now be called **BloatynosyAI** to emphasize the new Windows character. In addition to the ability to remove AI features in Windows, it also underscores the intelligent approach of the app itself. **This version works now on Windows 10 and also supports localizations.** If you want to translate the app, feel free to join the discussion here. 

All modules have been translated and are also available in BloatynosAI. Some are in a slightly streamlined form, while others are more powerful and extensible. 

**How does it work?**
The three central modules (Privacy, Bloatware Uninstaller, and App Installer) that we know from Bloatynosy are presented directly on the homepage. Plugins can be accessed, for example, via the keyword "Plugin" or "AI" to remove AI in Windows 11/10. Additionally, many other Windows functions can be accessed and controlled through the search bar. Just give it a try with some queries or keywords.

**Try out the new version and provide feedback in the discussions.**

![stclXTUXE0](https://github.com/builtbybel/Bloatynosy/assets/57478606/b5f67fa7-a4db-43b1-9a7a-e073d089e634)

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Bloatynosy/releases/tag/2.0'>BloatynosyAI 2.0</a>.</em>

**Comment by grzeegorz9020** [comment]
Pojawia się błąd.
![Bez tytułu](https://github.com/builtbybel/Bloatynosy/assets/48372871/e4877bd7-cf03-4c13-a490-5dd3a7dff7d6)

**Reply by Belim** [reply]
Which file is being sought? Does that emerge from the above exception, as I do not speak this language?

**Reply by Belim** [reply]
Do you have WebView2 runtime installed on your system? Or, to phrase it differently, is the Edge browser uninstalled?

**Reply by grzeegorz9020** [reply]
Unfortunately, after copying to C, it still doesn't work.

**Reply by grzeegorz9020** [reply]
I don't have Edge, it's deleted. I have firefox. I can download and install WebView2 only which 64 or 86 bit? My system is Windows 11 x64.

**Reply by Belim** [reply]
Yes, the WebView2 x64 runtime is a requirement, https://developer.microsoft.com/en-Us/microsoft-edge/webview2/?form=MA13LH#download

**Reply by Belim** [reply]
Uninstalling the Edge browser typically has no impact on the WebView2 runtime. In essence, the application should start without any issues. I'm not aware of how you managed to uninstall Edge or the WebView2 runtime.

**Comment by evaggelosevaggelou** [comment]
Just perfect. How can I ask questions in Greek;

**Reply by Belim** [reply]
It would need to be translated first.

**Comment by Vincentfaure66** [comment]
Sur la page GitHub de la dernière version, téléchargez le fichier Langfile_French.zip, décompressez l'archive .zip puis copiez les deux fichiers qu'elle contient dans le dossier BloatynosyAI > App, en choisissant de remplacer les fichiers existants par les nouveaux.

**Comment by Rateriu** [comment]
hello plz help I get this error when try to use  - Search crapware database (optional)

![Screenshot (10)](https://github.com/builtbybel/BloatynosyAI/assets/160390399/705c7ba9-d45c-49aa-b43b-a60d4a5fb815)
[!] Error loading crapware database: Could not find a part of the path 'C:\Users\antan\AppData\Local\Temp\Rar$EXa7428.2318\app\appxData.json'.
Ava says: I have found the following crapware. (If I present a false positive, please report it to my friend @Belim).

**Reply by Belim** [reply]
Unpack the archive beforehand.
